create/how_to/aws.md

```diff
@@ -98,11 +98,11 @@ Once you've downloaded your key pair (and only then), click on **Launch Instance
 
 Your instance may take a minute or two to get up and running (see the **Instance state** column).
 
-![Launch](/aws/09.launch.png)
+![Launching the instance](/aws/09.launch.png)
 
 Once the instance is ready, use your web browser to navigate to the **Public IPv4 address** or the **Public IPv4 DNS**. You should see the MeiliSearch [web interface](/reference/features/web_interface.md).
```

Comment:
```suggestion
Once the **Instance state** is **Running**, use your web browser to navigate to the **Public IPv4 address** or the **Public IPv4 DNS** displayed in your AWS instances dashboard. You should see the MeiliSearch [web interface](/reference/features/web_interface.md).
```

Reply:
Without this text, non-sighted people have no way to know what value they should wait for in the "Instance state" column.